setup.cfg

```diff
@@ -9,5 +9,6 @@ line_length = 88
 multi_line_output = 3
 use_parentheses = True
 lines_after_imports = 2
-known_third_party = PIL,bs4,classy_vision,cv2,detectron2,fairscale,faiss,fvcore,hydra,mock,nbconvert,nbformat,numpy,omegaconf,parameterized,pkg_resources,pycocotools,recommonmark,run_distributed_engines,scipy,setuptools,sklearn,sphinx_rtd_theme,tabulate,torch,torchvision,tqdm,utils
+ensure_newline_before_comments = True
+no_lines_before = FIRSTPARTY
```

Comment:
I would recommend to not introduce the new rule in this PR.

Reply:
I added these rules to minimize the effect the upgrade had on the existing files.